src/app/models/types/onwardJourney.ts

```diff
@@ -44,6 +44,19 @@ export type CpsRecommendation = {
   type?: string;
 };
 
+export type RecommendationNew = {
```

Comment:
This name is temporary to maintain compatibility with the old recommendations component. It will be renamed to `Recommendation` once all services get the new recommendations component.